src/__test__/_components/AddAssetContainer.test.js

```diff
@@ -18,6 +18,10 @@ const props = {
   createAsset: async () => {},
   loading: false,
   buttonLoading: false,
+  messages: {
```

Comment:
Hmm...shouldn't the status messages be on the same level as other props, not being properties of `messages`?

Reply:
Sorry, forgot to edit this.